python_functionality/homework.py

```diff
@@ -11,45 +11,58 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
     Make all `names` field in list of students to start from upper letter
 
     Examples:
-        fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
+        fix_names_start_letters([{'name': 'Alex', 'age': 26},
+        {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    for student in data:
+        if student.get('name'):
+            student['name'] = student['name'].capitalize()
+    return data
 
 
 def task_2_remove_dict_fields(data: DT, redundant_keys: List[str]) -> DT:
     """given_data
     Remove from dictionaries given key value
 
     Examples:
-       remove_dict_field([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 'age')
+       remove_dict_field([{'name': 'Alex', 'age': 26},
+       {'name': 'denys', 'age': 89}], 'age')
         >>> [{'name': 'Alex'}, {'name': 'denys'}]
     """
-    pass
+    new_data = []
+    for dict_instance in data:
+        for key in redundant_keys:
+            if key not in dict_instance.keys():
+                new_data += dict_instance
+    return new_data
 
 
 def task_3_find_item_via_value(data: DT, value) -> DT:
     """
     Find and return all items that has @searching value in any key
     Examples:
-        find_item_via_value([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 26)
+        find_item_via_value([{'name': 'Alex', 'age': 26},
+        {'name': 'denys', 'age': 89}], 26)
         >>> [{'name': 'Alex', 'age': 26}]
     """
-    pass
+    for dict_instance in data:
+        if value in dict_instance.values():
+            return [dict_instance]
 
```

Comment:
are you sure about that? What if few instances have the same value? Will it work properly?